nsqlbproducer/lbproducer.go

```diff
@@ -20,6 +20,11 @@ const (
 	RandomStrategy
 )
 
+const (
+	PublishNoTimeout      = time.Duration(-1)
```

Comment:
Do we really want to let the possibility to have no timeout? I don't see any case we want that..

Reply:
i'm not sure but since it's not the default, i thought that it could be handy.
I can remove it if needed.

Reply:
I would remove it, it's a bad practice not having a timeout :)